menu.go

```diff
@@ -3,24 +3,28 @@ package fyne
 // Menu stores the information required for a standard menu.
 // A menu can pop down from a MainMenu or could be a pop out menu.
 type Menu struct {
-	Label string
-	Items []*MenuItem
+	Label            string
+	Items            []*MenuItem
+	AfterNativeMenus bool
 }
 
 // NewMenu creates a new menu given the specified label (to show in a MainMenu) and list of items to display.
 func NewMenu(label string, items ...*MenuItem) *Menu {
-	return &Menu{label, items}
+	return &Menu{Label: label, Items: items}
 }
 
-// MenuItem is a sligne item within any menu, it contains a dispay Label and Action function that is called when tapped.
+// MenuItem is a single item within any menu, it contains a display Label and Action function that is called when tapped.
 type MenuItem struct {
-	Label  string
-	Action func()
+	Label             string
+	KeyEquivalent     string
```

Comment:
This is mac-specific terminology isn't it? It would be much better if we could use our Shortcut types...

Reply:
Probably, that was the lazy developer approach :). What about leaving it this way until I do the short-cut support (which then would include Fyne default short-cuts available in menus) which probably would show what is needed here anyhow?

Reply:
Or hold back on the shortcut portion of the PR for now?

Reply:
ack